lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss

```diff
@@ -4,16 +4,15 @@
 
 
 // Page Specific Components
-.campaign--action .container--prove,
+.campaign--action .container--prove {
+  background: $yellow;
+}
+
 .campaign--sms .container--do {
   background: $purple asset-url("images/campaign-pattern-default.png") repeat center 63px;
```

Comment:
Must just be getting overridden then: https://github.com/DoSomething/dosomething/blob/4e88027df55d9dbb38e3341b4e19f6899109634a/lib/themes/dosomething/paraneue_dosomething/images/campaign-pattern-default.png

Worth removing the rule, and also that image while you're at it!